media/js/firefox/hello/start.js

```diff
@@ -296,6 +314,10 @@
         // URL query param populated at template level in the view
         var incomingConversation = $main.data('incomingConversation');
 
+        // Get referrer and set Custom Variable. none is okay here.
+        var tourSource = $main.data('tourSource');
+        gaTrack(['_setCustomVar', 13, 'Hello FTU Referral', tourSource, 2]);
```

Comment:
Ahhh yes. I agree with you. We should only be looking for/setting this value if we're at the "Get Started" step.